src/locales/en.json

```diff
@@ -9,6 +9,7 @@
     "copy": "Copy",
     "copyAsPng": "Copy to clipboard as PNG",
     "copyAsSvg": "Copy to clipboard as SVG",
+    "copyAllTextNodesAsText": "Copy to clipboard as a single text element",
```

Comment:
`as single text element` doesn't make much sense (element). I'd change it to `Copy to clipboard as text`. Not sure if that much clearer though.